python_benchmarks/global_params.py

```diff
@@ -44,19 +50,35 @@
 
 # Utility function to generate input sizes for benchmarks
 def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
+    """
+    The weekly vs nightly input ranges only differ for 2D inputs currently.
+    Nightly input range:
+        Batch size: [16->16384] Hidden size: [768, 4*18432] (step size = 256)
+    Weekly input range:
+        Batch size:
+            [16]: Latency bound state
+            [512, 1024]: Just filled the machine
```

Comment:
what's the meaning of `just filled the machine`?

Reply:
Just filled the machine -> 1 full wave.
From the SOL vs hidden size graphs for different batch sizes, these are the batch sizes where we just start to be bandwidth bound instead of latency bound.